contrib/deploy-onefuzz-via-azure-devops/get_latest_version.py

```diff
@@ -34,8 +41,11 @@ def download_artifacts(self, path, artifacts):
         for artifact in artifacts:
             self.download_artifact(path, artifact["id"], artifact["name"])
 
-    def onefuzz_release_artifacts(self, path):
-        release_id, _ = self.get_latest_version()
+    def onefuzz_release_artifacts(self, path, version):
+        if version is not None:
+            release_id = self.get_release_id_by_name(version)
+        else:
+            release_id = self.get_release_id_by_name()
```

Comment:
`get_release_id_by_name` has assigned `None` by default. And `version` over here is either `None` or some value based on callstack. I think we don't need `if-else` statement over here.

Reply:
This if statement is redundant, so I've removed it all together.